### ISSUE TITLE
Playing TTS Stream Chunks

### ISSUE THREAD

**Issue by TomExMachina** [Shaunwei/RealChar#278]
I see in your code you are using the elevenlabs stream option, and sending chunks to the client via websockets. However, it seems like (By testing) that your web player is not beginning playback until the entire audio clips from elevenlabs has been received by the client. Am I correct about that? I am trying to solve the same problem.

**Comment by pycui** [comment]
yes. The chunk we are using it already a subset of the full response, so not a big issue. You can change to other streaming-based audio output web API -- feel free to contribute if you find it work
